### PR TITLE
CTCP-3511: Update PPNS schema

### PR DESCRIPTION
**Pull request by drnaylor-oc**

### CODE REVIEW

File: source/documentation/developer-setup.html.md.erb [hunk 1]
```diff
@@ -202,13 +202,30 @@ Message received push notifications must conform to the following JSON schema.
...
     "notificationType"
   ],
   "properties": {
     "messageUri": {
       "type": "string",
       "description": "URI of the message."
     },
+    "departureId": {
+      "type": "string",
+      "description": "ID of the movement if it's a departure."
```
Comment (fmcgrath) [inline source/documentation/developer-setup.html.md.erb:215]: [GDS style guide](https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style) advises against using contractions.

Suggestion:
ID of the movement if it is a departure.

File: source/documentation/developer-setup.html.md.erb [hunk 2]
```diff
@@ -202,13 +202,30 @@ Message received push notifications must conform to the following JSON schema.
...
+    },
+    "arrivalId": {
+      "type": "string",
+      "description": "ID of the movement if it's an arrival."
```
Comment (fmcgrath) [inline source/documentation/developer-setup.html.md.erb:219]: ID of the movement if it is an arrival.

File: source/documentation/developer-setup.html.md.erb [hunk 3]
```diff
@@ -309,6 +333,18 @@ Submission notification push notifications must conform to the following JSON sc
...
```
Comment (fmcgrath) [inline source/documentation/developer-setup.html.md.erb:342]: ID of the movement if it is an arrival.

**Review by fmcgrath — CHANGES_REQUESTED** [review]
No major issues. A few contractions need to be changed to meet style guidelines.

File: source/documentation/developer-setup.html.md.erb (hunk 1 above)
Comment (prashanthmamidi) [reply source/documentation/developer-setup.html.md.erb]: done

File: source/documentation/developer-setup.html.md.erb (hunk 2 above)
Comment (prashanthmamidi) [reply source/documentation/developer-setup.html.md.erb]: done

File: source/documentation/developer-setup.html.md.erb [hunk 4]
```diff
@@ -309,6 +333,18 @@ Submission notification push notifications must conform to the following JSON sc
       "type": "string",
       "description": "URI of the message."
     },
+    "departureId": {
+      "type": "string",
+      "description": "ID of the movement if it's a departure."
```
[the remark this answers is not in the document]
Comment (prashanthmamidi) [reply source/documentation/developer-setup.html.md.erb]: done

File: source/documentation/developer-setup.html.md.erb (hunk 3 above)
Comment (prashanthmamidi) [reply source/documentation/developer-setup.html.md.erb]: done

**Review by fmcgrath — APPROVED** [review]
Looks OK.

**Pull request merged**